pkg/controllers/user/certsexpiration/certsexpiration.go

```diff
@@ -161,3 +169,32 @@ func getCertExpiration(c string) (v3.CertExpiration, error) {
 		ExpirationDate: date.Format(time.RFC3339),
 	}, nil
 }
+
+//deleteUnusedCerts removes unused certs and cleans up kubelet certs when GenerateServingCertificate is disabled
+func deleteUnusedCerts(certsExpInfo map[string]v3.CertExpiration, rancherKubernetesEngineConfig *v3.RancherKubernetesEngineConfig) {
+	logrus.Info("Checking and deleting unused certificates")
```

Comment:
Add more info to this message, if will be too generic in a large setup, clusterID or other identification info.